core/viewer-wicket-impl/src/main/java/org/apache/isis/viewer/wicket/viewer/integration/wicket/ConverterForObjectAdapterMemento.java

```diff
@@ -54,8 +55,11 @@ public ObjectAdapterMemento convertToObject(final String value, final Locale loc
             return null;
         }
         final Oid oid = RootOid.deStringEncoded(value);
-        final ObjectAdapter adapter = getPersistenceSession().getAdapterFor(oid);
-        return ObjectAdapterMemento.createOrNull(adapter);
+        
```

Comment:
Method appears not used to me, but not really sure!

Reply:
Not sure either. 

IConverter<T> is part of Wicket's SPI, to convert strings that are input in forms to underlying objects.  So this one seems to convert a string representation of an object (eg "cust.Customer:123", like a Bookmark#toString() would return) into an ObjectAdapterMemento, which is basically the Wicket UI's serializable model for an ObjectAdapter (more or less the same as a RootOid).

So, my guess is that it isn't used, because I can't think of anywhere that we have this abiliity for a user to enter "cust.Customer:123" and have the framework look up the corresponding object reference.

Therefore, let's not worry about it too much.  Perhaps add a TODO to see about removing this entire class.